### DISCUSSION TITLE
Fly view Manipulator

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1892] [Support Q&A] [answered]
Hello,
How can I override the 90 degree pitch up limit of earth manipulator? Or Is there a fly view manipulator in osgEarth?
Is there a way to render a custom screen image directly on the screen as 2D. For example in Java a custom buffered image can be directly rendered on the screen. I am trying to create the same HUD, which is attached ,I did on NASA WorldWind Java.

Thanks

![image](https://user-images.githubusercontent.com/5169314/144735520-6a62d54a-f928-4ed9-8cb0-f8fe03469a71.png)

**Comment by gwaldron** [comment, marked as answer]
One question per thread, please :)

You cannot override the pitch limit. OE only has the one EarthManipulator that is designed for GIS-style navigation. The correct solution is to implement a CameraManipulator for your application. OpenSceneGraph has several to choose from -- you can adapt one of those or write your own.

As for the HUD, refer to the "osghud" example in OpenSceneGraph for an example of an overlay camera.
https://github.com/openscenegraph/OpenSceneGraph/blob/master/examples/osghud/osghud.cpp

Hope this helps.

**Reply by nebsar** [reply]
Thank you Glenn. I'll check it.
